tensorboard/webapp/deeplink/hash.ts

```diff
@@ -23,29 +22,24 @@ const TAB = '__tab__';
 
 @Injectable()
 export class HashDeepLinker implements DeepLinkerInterface {
-  private readonly tfStorage: TfStorageElement;
-
   constructor() {
-    this.tfStorage = document.createElement('tf-storage');
-    const tfGlobals = document.createElement('tf-globals');
-
     // Note: `migrateLegacyURLScheme()` must be called before `setUseHash`, so
     // that tfStorage reads from the actual URL, not the fake hash for tests
     // only.
-    tfGlobals.tf_globals.setUseHash(true);
-    this.tfStorage.tf_storage.migrateLegacyURLScheme();
+    window.tensorboard.tf_storage.migrateLegacyURLScheme?.();
```

Comment:
None of these funtions should be optional.

Reply:
Done. As per your other suggestion around the type declaration this is easy.